src/ddoc_cache/src/ddoc_cache_speed.erl

```diff
@@ -0,0 +1,61 @@
+-module(ddoc_cache_speed).
```

Comment:
Its marked as a TEMP commit. I'll end up removing it before merging the final PR. Its not a very rigourous test. To keep it in history I may remove it iwth a revert rather than a squash if you think it'd be worth it.